components/src/site-footer-condensed/site-footer-condensed.html

```diff
@@ -0,0 +1,79 @@
+<html>
+
+<head lang="en-US">
+	<meta http-equiv="X-UA-Compatible" content="IE=EDGE">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Site Footer | Component</title>
+	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+	<link rel="stylesheet" href="site-footer-condensed.css">
+	<script src="site-footer-condensed.js"></script>
+</head>
+
+<body>
+	<!-- COMPONENT:START -->
+	<div class="wsu-s-footer-condensed__wrapper">
+		<footer class="wsu-s-footer-condensed__container">
+
+			<div class="wsu-s-footer-condensed__contact-container">
+				<p class="wsu-s-footer-condensed__contact-info"><span class="wsu-s-footer__name">Lorem ipsum dolor sit amet</span>, PO Box 123456, <a href="tel:#" class="wsu-s-footer__contact-phone">P 509-123-4567</a>, <a href="tel:#" class="wsu-s-footer__contact-fax">F 509-123-4567</a></p>
```

Comment:
`wsu-s-footer__name` should be prefixed `wsu-s-footer-condensed__name` and bolder font style applied in css. Same with dark below.